components/lua/lua.h

```diff
@@ -19,7 +19,7 @@
 
 
 #define LUA_VERSION	"Lua 5.1"
-#define LUA_RELEASE	"Lua 5.1.4"
+#define LUA_RELEASE	"Lua 5.1.4" " on ESP-IDF v3.3.1"
```

Comment:
I dislike two things here:
- static version; in both the cloud build and the Docker image I extract it dynamically, see e.g. https://github.com/marcelstoer/nodemcu-custom-build/blob/master/ESP32/before-script.sh#L68
- the ESP-IDF version is unrelated to **LUA**_RELEASE; in my scripts I only append it to have it show up in the boot message
-> I would revert this line

Reply:
Whoops, that seems to have been accidentally added.  I blame my lack of `git`-fu.